docs/recon_configurations/README.md

```diff
@@ -345,6 +345,10 @@ class Transformation:
 ```
 transformations=[Transformation(column_name)="array_col",source=sort_array_input(array_col),target=sort_array_input(array_col)]
 ```
+> **Note:** `NULL` values are defaulted to `_null_recon` in the transformation expression.If User is looking for specific behaviour, they should tweak the rules in the transformation expression accordingly in these files: 1. [expression_generator.py](https://github.com/databrickslabs/remorph/tree/main/src/databricks/labs/remorph/reconcile/query_builder/expression_generator.py#L235) 2. [sampling_query.py](https://github.com/databrickslabs/remorph/tree/main/src/databricks/labs/remorph/reconcile/query_builder/sampling_query.py#L83)
```

Comment:
having line-numbers will not be efficient since when we update the code this will become stale, just mention descriptions and also add all the default transformation rules we have as a table.

Reply:
Updated, please check